src/apiProxy/apiConfigCreators.js

```diff
@@ -30,6 +30,34 @@ export function group(params) {
 }
 
 /**
+ * group communication endpoints
+ *
+ * accepts an `method` property in params
+ */
+export function mugcomm(params) {
+	const path = 'communications';
+	const { urlname, conversationId } = params;
+
+	let endpoint;
+	switch (params.method) {
```

Comment:
`method` already has a distinct meaning when making HTTP requests - probably better to find a different var name in this context. The underlying HTTP method will be forwarded from the initial request from the client.